prime-router/src/test/kotlin/fhirengine/translation/hl7/schema/converter/ConverterSchemaTests.kt

```diff
@@ -19,48 +18,45 @@ class ConverterSchemaTests {
         assertThat(schema.isValid()).isFalse()
         assertThat(schema.errors).isNotEmpty()
 
-        schema = ConverterSchema("ORU_R01")
+        schema = ConverterSchema("ca.uhn.hl7v2.model.v251.message.ORU_R01")
         assertThat(schema.isValid()).isFalse()
         assertThat(schema.errors).isNotEmpty()
 
-        schema = ConverterSchema("ORU_R01", "2.5.1")
+        schema = ConverterSchema("ca.uhn.hl7v2.model.v251.message.ORU_R01")
         assertThat(schema.isValid()).isFalse()
         assertThat(schema.errors).isNotEmpty()
```

Comment:
At this point I think this duplicates what we have at line 21 and can be removed.

Reply:
Very true. Removed